design/ceph/object/store.md

```diff
@@ -80,6 +80,7 @@ If there is a `zone` section in object-store configuration, then the pool sectio
 The gateway settings correspond to the RGW service.
 - `type`: Can be `s3`. In the future support for `swift` can be added.
 - `sslCertificateRef`: If specified, this is the name of the Kubernetes secret that contains the SSL certificate to be used for secure connections to the object store. The secret must be in the same namespace as the Rook cluster. Rook will look in the secret provided at the `cert` key name. The value of the `cert` key must be in the format expected by the [RGW service](https://docs.ceph.com/docs/master/install/ceph-deploy/install-ceph-gateway/#using-ssl-with-civetweb): "The server key, server certificate, and any other CA or intermediate certificates be supplied in one file. Each of these items must be in pem form." If the certificate is not specified, SSL will not be configured.
```

Comment:
Looks I need to update about k8s tls secrets here as well.

Reply:
1.6 rook docs contains secrets type https://rook.io/docs/rook/v1.6/ceph-object-store-crd.html#gateway-settings - that's why i did in the same way. But in master there are no such notes for some reasons.